R/EXADBI-info.R

```diff
@@ -0,0 +1,80 @@
+#' @include EXADBI-object.R
+#' @include EXADBI-driver.R
+#' @include EXADBI-result.R
+#' @include EXADBI-connection.R
+NULL
+
+#--------------------- dbGetInfo --------------------------
+
+#' @title dbGetInfo
+#' @describeIn dbGetInfo Returns metadata on a given EXAObject.
+#'
+#' @param dbObj An EXAObject.
+#' @return A named list.
+#' @export
+#' @author EXASOL AG <opensource@exasol.com>
+#' @seealso \code{\link[DBI:dbGetInfo]{DBI::dbGetInfo()}}
+setMethod(
+  "dbGetInfo", "EXAObject",
+  definition = function(dbObj) {
+    return("EXASOL DBI Object.")
```

Comment:
This does not return named list? Or is this a dummy method or an interface for the following methods?

Reply:
That's an overload of the dbGetInfo generic. As EXAObject is an empty base class there is no info to return.